tests/stub/routing.py

```diff
@@ -2756,11 +2795,15 @@ def router_script_with_one_reader_and_exit(self):
         return """
         !: BOLT #VERSION#
         !: AUTO RESET
-
         !: AUTO GOODBYE
+
         C: HELLO {"scheme": "basic", "credentials": "c", "principal": "p", "user_agent": "007", "routing": #HELLO_ROUTINGCTX# #EXTRA_HELLO_PROPS# }
         S: SUCCESS {"server": "#SERVER_AGENT#", "connection_id": "bolt-123456789"}
-        C: RUN "CALL dbms.routing.getRoutingTable($context, $database)" {"context": #ROUTINGCTX#, "database": "*"} {"[mode]": "r", "db": "system"}
+        {{
+            C: RUN "CALL dbms.routing.getRoutingTable($context)" {"context": #ROUTINGCTX#} {"[mode]": "r", "db": "system"}
+        ----
+            C: RUN "CALL dbms.routing.getRoutingTable($context, $database)" {"context": #ROUTINGCTX#, "database": "*"} {"[mode]": "r", "db": "system"}
```

Comment:
Is there some difference in how drivers invoke this? I thought that the previous version worked...

Reply:
I'm pretty sure, some drivers use the default db (can explicitly pass `null` or omit the database argument), while other's use the system db to do `verifyConnectivity`. Furthermore, this is just a missed change from the big PR #70. `router_script_with_one_reader_and_exit` in `Routing` has this line `C: ROUTE #ROUTINGCTX# [] "*"`, so this is the equivalent for Bolt 4.1. And finally, this script is used in `test_should_use_resolver_during_rediscovery_when_existing_routers_fail` and `test_should_successfully_acquire_rt_when_router_ip_changes` both seem not to be about testing the behavior of `verifyConnectivity`.

Reply:
As discussed in another chat, this is due to the difference in the way drivers implement the `verifyC` method. Please make sure this difference is captured somewhere in the cards so we can address it later on.